core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/locks/DatastoreLockIndexEntry.java

```diff
@@ -140,26 +139,47 @@ public Builder branchPath(final String branchPath) {
 			return this;
 		}
 		
+		public Builder timestamp(final Long timestamp) {
+			this.timestamp = timestamp;
+			return this;
+		}
+		
 		public DatastoreLockIndexEntry build() {
-			return new DatastoreLockIndexEntry(id, userId, description, parentDescription, repositoryId, branchPath);
+			return new DatastoreLockIndexEntry(id, userId, contexts, repositoryId, branchPath, timestamp);
 		}
 	}
 	
 	@ID
 	private final String id;
 	private final String userId;
-	private final String description;
-	private final String parentDescription;
 	private final String repositoryId;
 	private final String branchPath;
-	
-	private DatastoreLockIndexEntry(final String id, final String userId, final String description, final String parentDescription, final String repositoryId, final String branchPath) {
+	private final Long timestamp;
+	private final List<String> contexts;
+
+	// Only left in for backwards compatibility reasons
+	@Deprecated
+	private final String description = null;
```

Comment:
We currently don't throw away stuck lock entries during restart as we don't know which machine initiated it (yet), therefore there might be a case when there is a lock entry that uses the old format and the system would try to deserialize it but there is not setter in the JSON builder. I'll fix this soon.

Reply:
Fixed by [13be6a6](https://github.com/b2ihealthcare/snow-owl/pull/1353/commits/13be6a655c924a0bb41a69a6912a2c651df1cc7d)